api/views_export.py

```
@@ -4,22 +1,91 @@
+            }
+            for e in encounters
+        ],
+        "labs": [
+            {
+                "id": l.id,
+                "loinc": l.loinc,
+                "value": str(l.value),
+                "unit": l.unit,
+                "taken_at": l.taken_at.isoformat(),
+            }
+            for l in labs
```

Ruff (0.12.2)

Ambiguous variable name: `l`

(E741)